src/main/java/org/candlepin/subscriptions/subscription/SubscriptionSyncController.java

```diff
@@ -49,7 +50,7 @@
 @Component
 @Slf4j
 public class SubscriptionSyncController {
-  private static final int SUBS_PAGE_SIZE = 100;
+  private static final int SUBS_PAGE_SIZE = 1000;
```

Comment:
Previously we did larger-than-100-subs-per-page, but backed off to 100, as the time to complete 2000 subs would take over an hour. 1 hour is how long kafka will wait for a job to complete before declaring it stale, and would push the job back onto the queue.

Thanks to the query change used in CapacityReconcilliationController, the bottleneck is no longer in the database, but in the time to fetch subs from upstream. So we can fetch A LOT more subs and still not run into the 1 hour kafka message limit.

However, there's diminishing returns in fetching more than 1000 subs at a time, so we cap to that. Switching from 100-subs-per-page to 1000, for the test-org with lots of subs, improved fetch time up to 3x.